packages/@coorpacademy-components/src/molecule/select-multiple/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import map from 'lodash/fp/map';
+import pipe from 'lodash/fp/pipe';
+import join from 'lodash/fp/join';
+import filter from 'lodash/fp/filter';
+import get from 'lodash/fp/get';
+import ArrowDown from '@coorpacademy/nova-icons/composition/navigation/arrow-down';
+import TitledCheckbox from '../titled-checkbox';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+class SelectMultiple extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      opened: false
+    };
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.setSelect = this.setSelect.bind(this);
+  }
+
+  componentWillUpdate() {
+    const closeHandle = function(clickEvent) {
+      if (this.select && !this.select.contains(clickEvent.target)) {
+        this.setState({opened: false});
+      }
+    }.bind(this);
+    if (this.state.opened) {
+      document.addEventListener('click', closeHandle);
+      document.addEventListener('touchstart', closeHandle);
+    } else {
+      document.removeEventListener('click', closeHandle);
+      document.removeEventListener('touchstart', closeHandle);
```

Comment:
carrement des listeners sur le document ca me semble pas etre la bonne solution...
dautant plus que si ton component est unmount tu gardes les listeners.

faudrait plutot caler une div en fond en position fixed qd tu est state = opened par exemple ?

Reply:
@chrisdugne C'est pour fermer le select quand on click en dehors. Comment le fait d'ajouter une div en fond en position fixed qd le `state = opened` peut adresser cette problématique

Reply:
ta div recuperait le click justement;
en tout cas les listeners ne sont pas enlevés correctement là, met un console.log dans `closeHandle` et ouvre/ferme plusieurs fois tu verras..

Reply:
https://medium.com/@pitipatdop/little-neat-trick-to-capture-click-outside-react-component-5604830beb7f

--> la div "parent" et les listeners bien gérés aux mount/unmount